src/osd/ClassHandler.cc

```diff
@@ -75,9 +83,11 @@ int ClassHandler::open_all_classes()
 
 void ClassHandler::shutdown()
 {
-  for (auto& cls : classes) {
-    if (cls.second.handle) {
-      dlclose(cls.second.handle);
+  if (!disable_dlclose) {
```

Comment:
do we really need to check `disable_dlclose`? because `_load_class()` is never called if the cls are embedded in libcephd, `cls.second.handle` is not set. so this piece of code just works in both cases (embedded or not), am i right?